packages/ruby/spec/readme/payload_spec.rb

```diff
@@ -5,8 +5,11 @@
   it "returns JSON matching the readmeMetrics schema" do
     har_json = File.read(File.expand_path("../../fixtures/har.json", __FILE__))
     har = double("har", to_json: har_json)
-
-    result = Readme::Payload.new(har, {id: "1", label: "Anthony", email: "anthony@example.com"})
+    result = Readme::Payload.new(
+      har,
+      {id: "1", label: "Anthony", email: "anthony@example.com"},
+      true
```

Comment:
Thoughts on making the development flag into a keyword arg for better readability?

Reply:
👍